neondatabase/website
# pricing page update

This pull request brings update for pricing page.

TODO:

- [x] check actual links
- [x] check GA events

Steps to check:

- open [preview](https://neon-next-git-upd-pricing-page-neondatabase.vercel.app/pricing)
- make sure that everything looks and feels as expected
